patroni/config.py

```diff
@@ -257,7 +257,7 @@ def __init__(self, configfile: str,
 
               * file or directory path passed as command-line argument (*configfile*), if it exists and the file or
                 files found in the directory can be parsed (see :meth:`~Config._load_config_path`), otherwise
-              * YAML file passed via the environment variable (see :cvar:`PATRONI_CONFIG_VARIABLE`), if the referenced
+              * YAML file passed via the environment variable (see :attr:`PATRONI_CONFIG_VARIABLE`), if the referenced
```

Comment:
Hmm, in the class docstring it is declared as `:cvar:`. Shouldn't it be here the same?

Reply:
Apparently, it's not a valid "domain" according to sphinx command output. Docs are here for what can be used
https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-python-objects